### ISSUE TITLE
Support background gradients instead of flat color

### ISSUE THREAD

**Issue by Maijin** [radareorg/cutter#1378]
**Is your feature request related to a problem? Please describe.**

Support background gradients instead of flat color
From @radare 

**Describe the solution you'd like**

Add Support background gradients instead of flat color

**Comment by thestr4ng3r** [comment]
![giphy](https://user-images.githubusercontent.com/1460997/54865729-5f2a7180-4d6a-11e9-9f75-f2537dc07330.gif)

**Comment by a1ext** [comment]
Why not, IDA has such ability

**Comment by trufae** [comment]
Why closing my issues and reopining them again by you?

**Comment by Maijin** [comment]
@trufae we need the issues splitted

**Comment by scrapbird** [comment]
Haha how do you expect something like this to look nice though? And do you mean just in `VV` or in `v` too?

**Comment by trufae** [comment]
yeah i just recall that "feature" when reviewing some ida screenshots. obv no need to for transparencies or images like Eterm..  but making a small gradient between the selected color and a lighter one may look nice i guess. just a proposal, feel free to discard if u dont like it or its an unnecessary overengineering
